charms/trusty/hss-proxy/README.md

```diff
@@ -0,0 +1,16 @@
+# Overview
+
+This is a proxy [Juju charm](https://jujucharms.com/about) for the 3GPP-Cx interface to the HSS.  It does not provide HSS function.
```

Comment:
Worth a link or fuller explanation of what a proxy charm is? (Googling it isn't very helpful.)